### DISCUSSION TITLE
can i use this EFi for my  legion 5 15ACH6H?

### DISCUSSION THREAD

**Discussion by jithu12** [kalkmann/Legion-5600H-Hackintosh#27] [Q&A]
Hi Kalkmann,

I hope you're doing well. I came across your fantastic Repo on installing macOS on your Lenovo Legion laptop, and I'm very impressed with your work. I have a Lenovo Legion 5 15ACH6H with slightly different specifications and was wondering if you could help me with the process.

Here are the specs of my laptop:

**Model**: Lenovo Legion 5 15ACH6H
**Processor**: AMD Ryzen 7 5800H with Radeon Graphics
**RAM**: 32GB Crucial DDR4-3200
**Graphics Card 1**: AMD Radeon(TM) Graphics (1024 MB)
**Graphics Card 2**: NVIDIA GeForce RTX 3060 Laptop GPU (6 GB)
**SSD**: 512 GB + 1 TB
**Screen Resolution**: 2560 x 1440

Despite my best efforts to find forums or additional guides, I have not been able to find anything specific to my setup. Your repository has been a beacon of hope, and I would be incredibly grateful if you could provide any guidance or modifications necessary for my configuration, especially concerning the differences in graphics cards and screen resolution.

Thank you so much for your time and assistance. Your expertise would be incredibly helpful in getting macOS up and running on my machine.

Thnak you.

**Comment by kalkmann** [comment]
Thanks for the kind words, I'm just a curious guy.

On your machine, you will have to use the Nootedred kext to use Ryzen's integrated video. I've never tried it, but from the documentation, there are some graphical and other limitations. Depending on usage, it could be insignificant.

More one reference you could use about using Nootedred Kext on a Lenovo with 5800H (very close specifications):

https://github.com/zabdottler/Lenovo-Yoga-16S-hackintosh

Where you can get more help:

- https://forum.amd-osx.com/

- https://discord.gg/EfCYAJW

Good luck!

**Reply by jithu12** [reply]
Thank you Kalkmann,

**Reply by kalkmann** [reply]
Check this fork https://github.com/ducviet5138/Hackintosh-Legion-5
